
AUTOCONF_INCLUDED is not defined anymore in 2.6.38, so old include got exposed.

Fix by removing the offending bits.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

 3rdparty/acerhk/acerhk.c |    4 ----
 1 file changed, 4 deletions(-)

--- linux-2.6.38/3rdparty/acerhk/acerhk.c.orig	2011-03-20 13:57:32.000000000 +0200
+++ linux-2.6.38/3rdparty/acerhk/acerhk.c	2011-03-20 15:00:45.768659568 +0200
@@ -35,10 +35,6 @@
  *
  */
 
-#ifndef AUTOCONF_INCLUDED
-#include <linux/config.h>
-#endif
-
 /* This driver is heavily dependent on the architecture, don't let
  * anyone without an X86 machine use it. On laptops with AMD64
  * architecture this driver is only useable in 32 bit mode.
